.github/workflows/deploy.yaml

```diff
@@ -22,6 +22,21 @@ env:
   TAG: sha-${{ github.sha }}
 
 jobs:
+  is-tagged-release:
+    runs-on: ubuntu-latest
+    outputs:
+      tagVersion: ${{steps.tagVersion.outputs.tagVersion}}
+    steps:
+      - name: Git Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Is Tagged Commit
+        id: tagVersion
+        # Check if one of the commits is associated with a tag
+        run: |
+          echo ::set-output name=tagVersion::$( git tag --contains ${{ github.sha  }} | grep v*)
```

Comment:
Looks like this only checks the last commit that was pushed. This would require that we fast-forward the release branch. Although this is the process that is documented, I would suggest that we update that process (and the documentation in the readme) to allow users to click the github ui merge button. 
Looking at the [git docs](https://git-scm.com/docs/git-tag#Documentation/git-tag.txt---mergedltcommitgt), and confirmed with local testing, this should work: 
`git tag --merge ${{ github.sha  }} --no-merge ${{ github.event.before }}`

Reply:
In older version of git these two commands can't be combined. I'm sure github runners are using the latest version but it would be funny if it were an old enough version that this was not compatible with the use of this command